proto/storage/cve.proto

```diff
@@ -128,6 +136,8 @@ message ImageCVE {
   bool snoozed = 7 [deprecated = true]; // @gotags: search:"CVE Snoozed"
   google.protobuf.Timestamp snooze_start = 8 [deprecated = true];
   google.protobuf.Timestamp snooze_expiry = 9 [deprecated = true]; // @gotags: search:"CVE Snooze Expiry,hidden"
+  float nvdcvss = 10; // @gotags: search:"NVD CVSS",store"
```

Comment:
After discussions with the product team, we identified the need to address cases where the NVD score is unavailable. For vulnerability reports generated by the Scanner, if the NVD score is missing, the NVD CVSS metric will not appear in the report.
Therefore, during the translation process from the vulnerability report, we need to clearly indicate when NVD data is unavailable.

Reply:
@daynewlee  I have created a ticket https://issues.redhat.com/browse/ROX-26097 for this. that translation will be a new pr and not part of this one

Reply:
@ksurabhi91 are you planning on handling the translation in the API layer?

Reply:
@dashrews78  that will be a separate pr from scanner. here is the  ticket https://issues.redhat.com/browse/ROX-26097